src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -546,6 +587,21 @@ class ContentSidebar extends PureComponent<Props, State> {
         }
     }
 
+    /**
+     * Fetches a Users info
+     *
+     * @private
+     * @param {string} [id] - User id. If missing, gets user that the current token was generated for.
+     * @return {void}
+     */
+    fetchCurrentUser(id?: string = '', shouldDestroy?: boolean = false): void {
+        if (shouldRenderSidebar(this.props)) {
```

Comment:
Why is this if check needed here?

Reply:
Following suit with other API requests. No reason to re-render all of these components if nothing is being shown